src/partials/layout.html

```diff
@@ -221,5 +221,54 @@
       });
     </script>
     <script async src="https://cdn.announcekit.app/widget-v2.js"></script>
+    <script>
+      filterSelection("all")
+      function filterSelection(c) {
+        var x, i;
+        x = document.getElementsByClassName("filterDiv");
+        if (c == "all") c = "";
+        // Add the "show" class (display:block) to the filtered elements, and remove the "show" class from the elements that are not selected
+        for (i = 0; i < x.length; i++) {
+          RemoveClass(x[i], "show");
+          if (x[i].className.indexOf(c) > -1) AddClass(x[i], "show");
```

Comment:
The easiest way to add/remove element classes is via [`element.classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList).

In this case you can add attribute like `data-type="language"` to the div elements and with that we can just do `x.classList.toggle("hidden", x.dataset.type === c || c === "all")`. Let's also rename `c` to `integrationType`, or similar :)